testing/Dockerfile.querysdk.test

```diff
@@ -0,0 +1,17 @@
+FROM node:19.6.1-slim@sha256:a1ba21bf0c92931d02a8416f0a54daad66cb36a85d2b73af9d73b044f5f57cfc
+
+RUN apt-get update && apt-get -y install \
+  git python3 make curl netcat
+
+RUN mkdir -p /app
+WORKDIR /app
+
+COPY sdk/js-query/package.json sdk/js-query/package-lock.json ./sdk/js-query/
```

Comment:
Do we need this `COPY` since we're also copying the `sdk/js-query` directory below?

Reply:
TIL